test/test_common/simulated_time_system_test.cc

```diff
@@ -35,7 +35,7 @@ class SimulatedTimeSystemTest : public testing::Test {
   }
 
   void sleepMsAndLoop(int64_t delay_ms) {
```

Comment:
If we're adding functionality can we add tests too?

Reply:
not following you; what functionality is not covered?

Reply:
Actually I see what you were getting at I think, added test AdvanceTimeWait.